twill-api/src/main/java/org/apache/twill/api/ResourceReport.java

```diff
@@ -61,4 +63,19 @@
    * @return list of services of the application master.
    */
   List<String> getServices();
+
+  /**
+   * Get the map of the log level arguments of the twill application.
+   *
+   * @return the map of the log level arguments of the twill application
+   */
+  Map<String, Map<String, LogEntry.Level>> getLogLevelArguments();
```

Comment:
The key of the map is the runnable name, and the value of the map is a map of logger name and log level pair. If nothing is set for log level, it will return an empty map.